README.md

```diff
@@ -77,7 +77,7 @@ Using the gem https://github.com/clio/ten_years_rails to help with the upgrade p
 ```
 docker-compose -f docker-compose-rails-next.yml build
 
-docker-compose -f docker-compose-rails-next.yml run --service-ports --rm talkapi bash
+docker-compose -f docker-compose-rails-next.yml run --service-ports --rm -e RAILS_ENV=test talkapi bash
```

Comment:
Not sure about this...this env var should only be added if you intend to run a console in the test env, which I don't often do. I use this when I'm testing the API and want console access to see whatever I just did. Maybe add a test section?

Reply:
Ah. i should be more descriptive on this. Updated: 
1)) Update the title of this section to use docker-compose-rails-next vs docker-compose to make it clear this is next 
2) Add description that this is to bash in so tests can run. (A command run a lot while upgrading rails/fixing tests)